src/server/scripts/World/mob_generic_creature.cpp

```diff
@@ -27,7 +27,10 @@ EndScriptData */
 #include "ScriptedCreature.h"
 #include "PassiveAI.h"
 
-#define GENERIC_CREATURE_COOLDOWN   5000
+enum GenericCreature
+{
+    GENERIC_CREATURE_COOLDOWN = 5000
```

Comment:
OK, so I will drop both the enum and #define completely and just insert `std::chrono pls (`Seconds(5)`)` at each occurrence in the code instead. Very well. Thanks for the tip. :+1: